core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/kubernetes/KubernetesContainer.scala

```diff
@@ -104,7 +104,8 @@ class KubernetesContainer(protected[core] val id: ContainerId,
     super.suspend().flatMap(_ => kubernetes.suspend(this))
   }
 
-  def resume()(implicit transid: TransactionId): Future[Unit] = kubernetes.resume(this)
+  override def resume()(implicit transid: TransactionId): Future[Unit] =
+    kubernetes.resume(this).map(_ => super.resume())
```

Comment:
This should be a `flatMap` to also "wait" for the future returned by super.resume.